evm/src/cpu/kernel/asm/hash/sha2/message_schedule.asm

```diff
@@ -3,32 +3,35 @@
     // stack: num_blocks
     %mul_const(64)
     %add_const(2)
+    PUSH @SEGMENT_KERNEL_GENERAL
+    GET_CONTEXT
+    %build_address
 %endmacro
 
 // Precodition: stack contains address of one message block, followed by output address
 // Postcondition: 256 bytes starting at given output address contain the 64 32-bit chunks
 //                of message schedule (in four-byte increments)
-gen_message_schedule_from_block:
+global gen_message_schedule_from_block:
```

Comment:
Are these added `global` labels necessary? (This one and the ones below, in the same file)

Reply:
They were necessary for debugging when I had messed something up 🫣 will revert!

Reply:
And I removed one too many global label 🤦🏼 Last commit adds it back